test/Microsoft.Health.Dicom.Web.Tests.E2E/Rest/ChangeFeedTests.cs

```diff
@@ -27,6 +27,7 @@ public ChangeFeedTests(HttpIntegrationTestFixture<Startup> fixture)
             _client = fixture.Client;
         }
 
+        [Trait("Category", "paas")]
```

Comment:
We might want to call these something more broadly appropriate. Like if these tests are useful to check the high level functionality of the service, maybe something like "smoke-test" or "bvt"